force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -139,15 +196,19 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static String returnHelloWorld() {
         // Initialize the result String
+        String strHelloWorld = 'Hello World!';
+        String result = '';
 
         // Use a for loop to append 'Hello World!;' to the result string three times
-        /* 
-         for (){
-            
+         
+         for (Integer i = 0; i < 3; i++){
+            result += strHelloWorld;
+            if(i <3){
+                result += '; ';
+            }
```

Comment:
Good but  no need to add the semi colon separately in this one. 

Below is my solution

```
    public static String returnHelloWorld() {
        String result = '';
        Integer count = 0;
        while(count < 3){
            result += 'Hello World!; ';
            count++;
        }
        return result; // Replace null with the variable you used to store the result
    }
```